dev.php

```diff
@@ -187,6 +187,13 @@ function shortcode_ui_dev_advanced_example() {
 			'multiple' => true,
 		),
 		array(
+			'label'    => esc_html__( 'Select Term', 'shortcode-ui-example' ),
```

Comment:
CS: escape late on output, so only translation should be here. `__()` instead of `esc_html__`